emscripten.py

```diff
@@ -452,22 +452,11 @@ def finalize_wasm(infile, outfile, memfile, DEBUG):
   else:
     args.append('--no-legalize-javascript-ffi')
   if memfile:
-    args.append('--separate-data-segments=' + memfile)
+    args.append(f'--separate-data-segments={memfile}')
+    args.append(f'--global-base={shared.Settings.GLOBAL_BASE}')
```

Comment:
Are there any plans to get rid of this usage as well?

Reply:
Sounds good but I'm not sure if its feasible.  

@azakai wrote this comment on `separateDataSegments`:

```
// Emits the data segments to a file. The file contains data from address base
// onwards (we must pass in base, as we can't tell it from the wasm - the first
// segment may start after a run of zeros, but we need those zeros in the file).
```

`GLOBAL_BASE` is almost always fixed 1024 on emscripten and wasm-ld both.   Having it exist as a build time thing seems OK to me... I think its already been removed generated JS.